db/citations.yml

```diff
@@ -10783,3 +10783,28 @@ zhu;gca11:
   wwwemail: ''
   wwwpub: http://slapper.apam.columbia.edu/bib-eu9iifae/papers/12zhugca.pdf
   year: '2012'
+gu;ieeesp21:
+  ackno: "This research was supported by the US National Science Foundation (NSF) through
```

Comment:
copy-paste the full acknowledgement from the paper.  I believe it says things like QD acknowledges this and that....etc.